charts/darts-api/values.yaml

```diff
@@ -16,6 +16,12 @@ java:
           alias: DARTS_API_DB_HOST
         - name: AppInsightsInstrumentationKey
           alias: APP_INSIGHTS_KEY
+        - name: Azure-AD-B2C-TenantIdKey
```

Comment:
keys are named something different in the vault:-

AzureAdB2CClientIdKey
AzureAdB2CClientSecretKey
AzureAdB2CTenantIdKey